simulator/setting_up_simulator.md

```diff
@@ -6,118 +6,76 @@ title: Setting up the Simulator
-Your Python path is likely similar to `/Library/Frameworks/Python.framework/Versions/3.11/bin/python3` when using Python 3.11.
-If you installed Python using Homebrew, it will be in `/opt/homebrew/bin/python3.11` (or `/usr/local/bin/python3.11` on an Intel-based Mac)
+- The `simulator` folder contains our code to support running your code in the simulator.
+- The `zone_0` folder is where you will write your code, and it must contain a file called `robot.py`.
+- The `setup.py` and `run_simulator.py` files are used to set up the environment and run the simulator respectively.
+- The `readme.html` file contains a single page guide to using the simulator, similar to this one.
 
-If you're still having problems, ask for help in [Discord][discord].
+If the world supports multiple zones, you will see a `zone_1` folder, and so on.
 
+<div class="info" markdown="1">
+If you had previously downloaded the simulator, you can copy your code from the previous installation by copying just the `zone_0` folder from the old installation to the new one.
+</div>
 
-## The world simulation {#world-simulation}
+## Setting up the Environment
 
-The world simulation we provide defines the environment that contains the simulated arena and robot as well as the SR API for use in the simulator.
+Now that you have downloaded and extracted the simulator, you need to set up the environment to run the simulator.
+Since the simulator uses the sr-robot3 library, there are a series of python packages that need to be installed and Webots needs to be configured to use the correct version of Python.
+We have provided a script that will set up this environment for you.
 
-1. Create a directory, perhaps called `simulation` where you will store your robot code.
-2. [Download the simulation](https://github.com/srobo/competition-simulator/releases/download/sr2024.2/competition-simulator-sr2024.2.zip), the latest version is sr2024.2, released on 2024-02-04.
-3. Unzip the simulation as a folder inside the folder you created in the first step:
-    ```
-    simulation
-    ├── competition-simulator-<version>
-    │   ├── ...
-    │   └─ libraries.txt
-    │   └─ worlds
-    │       └── Arena.wbt
-    └── robot.py
-    ```
-    If there is not an existing `robot.py` an example one will be created when the simulator first runs.
+First, navigate to the folder you extracted the simulator into. This folder should contain a file called `setup.py`.
+Run this script and it will set up the environment for you.
+A terminal window will open and you will see the output of the script, if there are any errors displayed you should ask for help in [Discord][discord].
 
-4. Open the Webots IDE, then use that to open the `worlds/Arena.wbt` file.
+<div class="info" markdown="1">
+In order to run the Python script, instead of opening the file you may need to right-click and select **Open with** &rarr; **Python**.
```

Comment:
What's the mac equivalent to "Right click > Open With..."?

Reply:
The same.